modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -185,6 +185,90 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-145665 - Verify it's possible to add an object entry with the trigger On Order Status Update"
+	@priority = "4"
+	test CanCreateActionAddObjectEntryWithOnOrderStatusUpdate {
+		task ("Given an action in the Commerce Order triggered with Custom Object.") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 171861",
+				objectName = "CustomObject171861",
+				pluralLabelName = "Custom Objects 171861");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Field",
+				fieldType = "Text",
+				isMandatory = "true");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject171861");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Order");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Action Name",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "Value Test",
+				objectLabel = "Custom Object 171861",
+				thenAction = "Add an Object Entry",
+				whenAction = "On Order Status Update");
+		}
+
+		task ("When the user selects the trigger OnOrderStatusUpdate using the action Add Object Entry.") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Accounts Name Test",
+				accountEntryType = "Business");
+
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "1",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
+
+		task ("and When: The user changes the order status.") {
+			CommerceNavigator.gotoPortlet(
+				category = "Order Management",
+				portlet = "Orders");
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "Pending");
+
+			Click(
+				key_text = "Accept Order",
+				locator1 = "Link#ANY");
+		}
+
+		task ("Then the action is triggered on Object entries.") {
```

Comment:
Something like "Then the action is triggered to add an object entry" is good here to be more descriptive of the final assertion.